src/resources/PowerShellWorkerStrings.resx

```diff
@@ -1,166 +1,208 @@
-<?xml version="1.0" encoding="utf-8"?>
-<root>
```

Comment:
Can you please reformatting this file to make it less different from the original? Spaces characters were added to many places and cause it to look so differently. It's hard to review changes in this file.